index.html

```diff
@@ -4,9 +4,58 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Google</title>
+    <link rel="stylesheet" href="main.css">
 </head>
 <body>
+    <nav>
+        <div id="left">
+            <a class="left" href="#">About</a>
+            <a class="left" href="#">Store</a>
+
+        </div>
+        <div id="right">
+            <a class="gmail" href="#">Gmail</a>
+            <a href="#">Images</a>
+            <img id="google-app" src="./googleapp.png" alt="Google apps " title="Google apps">
+            <button type="submit" class="btn">Sign In</button>
+
+        </div>
+    </nav>
+     
```

Comment:
Please add all the code inside one main <div> to make it more semantic so that the page doesn't break when making any changes or update in the future. Thank you.